lib/react_client/js_interop_helpers.dart

```diff
@@ -59,7 +76,9 @@ dynamic jsifyAndAllowInterop(object) {
 }
 
 _convertDataTree(data) {
-  final _convertedObjects = new Map.identity();
+  // Use _jsObjectFriendlyIdentityHashCode instead of `identityHashCode`/`Map.identity()`
+  // to work around https://github.com/dart-lang/sdk/issues/47595
+  final _convertedObjects = LinkedHashMap(equals: identical, hashCode: _jsObjectFriendlyIdentityHashCode);
```

Comment:
`Map.identity()` is equivalent to 
```dart
LinkedHashMap(equals: identical, hashCode: identityHashCode)
```
so this essentially just changes `identityHashCode` to `_jsObjectFriendlyIdentityHashCode`